frontend/packages/console-shared/src/components/dynamic-form/fields.tsx

```diff
@@ -26,75 +30,144 @@ export const DescriptionField: React.FC<FieldProps> = ({ id, description }) =>
     </span>
   ) : null;
 
-const FieldWrapper: React.FC<FieldProps> = ({
+export const DescriptionField: React.FC<FieldProps> = ({ id, description }) => (
+  <Description id={id} description={description} />
+);
+
+export const FormField: React.FC<FormFieldProps> = ({
```

Comment:
Moved `FormField` component from templates file and use instead of `FieldWrapper` component since these two were very similar components. Moving it here was mostly to prevent dependency cycles.

Reply:
Also refined the props for this component so that it's more easily reused between field and template components.